ONSdigital/ras-frontstage
# RAS-1380 Integrate party new enrolment and respondent in frontstage

# What and why?
This PR integrates the new respondent and enrolment endpoints

# How to test?
The new respondent and enrolment endpoints are used in 15 places. To cover them all you need to
1. Login to an account with an enrolment
2. Go to my account and change your name
3. Go back to my account and change your email, and accept the change
4. Share access to a ce to a new respondent, and accept by the respondent
5. Share access to a ce to respondent that already exists and accept by the respondent
6. Transfer a survey (i.e I am no longer required to complete any surveys in my accounts) and accept by the respondent
7.  Open an survey for an eQ
8. Delete your account

Care must be taken when go from one account to another as the session will be wrong, it's best to logout in each instance. 

**N.B I have checked each scenario, it might be a case of allowing the business testers to do the manual checks and focus on the code**



# Jira
